content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,361 @@
+   changes.
+
+Now, specific removal strategies can be defined on product categories. To do this, go to
+:menuselection:`Inventory app --> Configuration --> Product Categories`, and choose a product
+category to define the removal strategy on. In the :guilabel:`Force Removal Strategy` field, choose
+a removal strategy.
+
+.. image:: removal/product-category-removal.png
+   :align: center
+   :alt: :alt: Removal strategy on a product category.
 
-Next, you need to define your removal strategy, on *Product Categories* via
-:menuselection:`Inventory --> Configuration --> Product Categories`.
+To view all products with lots/serial numbers assigned to them, navigate to
+:menuselection:`Inventory app --> Products  --> Lots/Serial Numbers`. This reveals a page with
+drop-down menus of all products assigned lots or serial numbers, filtered by *product* by default.
+To change the category these products are filtered by, click :guilabel:`Product` (in the search bar,
+in the top right of the page) to remove the default filter, and select a new filter if desired.
 
-.. image:: removal/first-in-first-out.png
+.. image:: removal/lot-serial.png
    :align: center
-   :alt: force removal strategy set up as first in first out.
+   :alt: Click on Products, then Lots/Serial Numbers to display all the products with lots or serial
+         numbers.
+
+To view the serial numbers being selected for a sales order, go to the :guilabel:`Sales app` and
+select the sales order in question. In the sales order, click the :guilabel:`Delivery` smart button
+in the top right. In the :guilabel:`Operations` tab, click the :guilabel:`⋮≣ (Detailed Operations)`
+icon in the far right for the product in question. The :guilabel:`Detailed Operations` window
+appears, and displays the lot or serial numbers selected for that specific product for the delivery
+order.
 
 FIFO (First In, First Out)
 --------------------------
 
-As said, a *FIFO* strategy implies that products stocked first move out first. Companies should use
-that method if they are selling products with short demand cycles, such as clothes, and to ensure
-they are not stuck with outdated styles in stock.
+The :abbr:`FIFO (First In, First Out)` removal strategy implies that products which enter a
+warehouse's stock first are removed first. Companies should use this method if they are selling
+products with short demand cycles, such as clothes, to ensure they are not stuck with outdated
+styles in stock.
 
-For this example, we created three lots of white shirts. Those are from the All/Clothes category,
-where we put *FIFO* as the removal strategy. In our stock location (WH/Stock), we now find the
-three lots available.
+In this example, there are three lots of white shirts. The shirts are from the All/Clothes category,
+where :guilabel:`FIFO` is set as the removal strategy. In the :guilabel:`Inventory Valuation
+Report`, the three different receipts are listed with the amounts.
```

Comment:
The first half of this example is not part of an active instruction, so using italics rather than guilabel